src/util/ConfigManager/ConfigManager.cpp

```diff
@@ -353,18 +483,36 @@ ConfigManager::getListOfNotChangedConfigOptionsWithDefaultValuesAsString()
 }
 // ____________________________________________________________________________
 void ConfigManager::verifyWithValidators() const {
+  // Check all the validators in sub managers.
+  std::ranges::for_each(
+      configurationOptions_,
```

Comment:
1. If you sprinkle in a `views::values` somewhere, then you get rid of the `get<1>` (can maybe also be done in other functions above).

2. Idea for a util (maybe on a different branch to not stall the merging):
I think in `util/TransparentFunctors` we already have `IsInstantiation` and `get` to here replace the `ifConstepxr` with ranges.
But I think we can build an even more powerful abstraction that filters a range of variants to receive only the elements that hold a certain type. (This is exactly what happens here, but we also have other occurences).

Reply:
> 1. If you sprinkle in a `views::values` somewhere, then you get rid of the `get<1>` (can maybe also be done in other functions above).

I don't follow. About which `get<1>` are you talking about? There is none here and all the other, that I can see in the current version, can't be replaced with `views::values`, because I need access to both the elements.

>        I think in `util/TransparentFunctors` we already have `IsInstantiation` and `get` to here replace the `ifConstepxr` with ranges.

I can't say, that I follow your line of thought.  Why would I replace `if constexpr` with a function combination, that only takes ranges? And what have `IsInstantiation` and `get` to do with it?

Reply:
1. I found the corresponding `get<0>` to your `get<1>`s. 
I suggest using structured bindings `const auto& [first, second] = ...` to make this more readable.

2. I meant the pattern that we have a range of variants, and only want to do something for one of the contained types, and ignore all the others. This pattern can be solved with ranges that hide the `std::visit` machinery, but as I said, probably this is for a different PR where we can talk about this separately.

Reply:
1. Good idea. Done in [0351672](https://github.com/ad-freiburg/qlever/pull/1071/commits/0351672d414770f4270047bb1093488539ecd950).

2. I'm not sure, if we do that pattern often enough, in order for such an implementation to be worthwhile.